src/OFS/Image.py

```diff
@@ -520,6 +520,9 @@ def _read_data(self, file):
         n = 1 << 16
 
         if isinstance(file, str):
+            raise ValueError("Must be bytes")
```

Comment:
Is this branch for Python 3 only? I ask because this seems wrong for Python 2: `str` and `bytes` are the same there, so in practice both `str` and `bytes` will get this ValueError.
Maybe do it like this:

```
if isinstance(file, bytes):
    ... handle normally
elif isinstance(file, str):
    raise ValueError("Must be bytes")
```

Or actually, since the top of the file has `unicode = str` when on Python 3, the `str` instance check can become:

```
if isinstance(file, unicode):
   raise ValueError("Must be bytes")
```

Note: I don't use Python 3 as much as I should like, so I may be wildly off base here.